api/src/main/java/org/pmiops/workbench/api/BigQueryService.java

```diff
@@ -49,17 +49,14 @@
 
   @VisibleForTesting
   protected BigQuery getBigQueryService() {
-    DbCdrVersion cdrVersion = CdrVersionContext.getCdrVersion();
-    if (cdrVersion == null) {
+    String projectId = CdrVersionContext.nullableGetBigQueryProject();
```

Comment:
This is the only situation where a null CDR Version is expected, so I special-cased it for safety